src/libs/ui/docsetsdialog.cpp

```
@@ -385,7 +386,8 @@ void DocsetsDialog::downloadProgress(qint64 received, qint64 total)
         QTemporaryFile *tmpFile = m_tmpFiles[docsetName];
         if (!tmpFile) {
             tmpFile = new QTemporaryFile(QStringLiteral("%1/%2.XXXXXX.tmp").arg(Core::Application::cacheLocation(), docsetName), this);
-            tmpFile->open();
+            if (!tmpFile->open())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)